saleor/static/dashboard-next/components/CategoryList/CategoryList.tsx

```diff
@@ -12,15 +12,20 @@ import i18n from "../../i18n";
 import Skeleton from "../Skeleton";
 
 interface CategoryListProps {
-  categories?: CategoryPropertiesQuery["category"]["children"]["edges"];
+  categories: Array<{
+    id: string;
+    name: string;
+  }>;
+  loading?: boolean;
   onClick?(id: string);
 }
 export const CategoryList: React.StatelessComponent<CategoryListProps> = ({
   categories,
-  onClick
+  onClick,
+  loading
 }) => (
   <List>
-    {categories === undefined ? (
+    {loading ? (
```

Comment:
Why though? If we have a cache, we should do the initial render from cache.

Reply:
I totally forgot about this!